applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -632,3 +653,13 @@ def _create_line_search_strategy(self):
                             self.settings["compute_reactions"].GetBool(),
                             self.settings["reform_dofs_at_each_step"].GetBool(),
                             self.settings["move_mesh_flag"].GetBool())
+
+    def _create_runge_kutta_4_strategy(self):
+        computing_model_part = self.GetComputingModelPart()
+        convection_diffusion_scheme = self.get_solution_scheme() # to call _create_solution_scheme method of the solver class
+        explicit_builder_and_solver = self.get_builder_and_solver()
+        rebuild_level = 0
+        return ConvectionDiffusionApplication.ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(computing_model_part,
+                            explicit_builder_and_solver,
+                            self.settings["move_mesh_flag"].GetBool(),
+                            rebuild_level)
```

Comment:
Why are you adding this to the base solver if you are creating your own explicit one?

Reply:
Moving to explicit solver.